### ISSUE TITLE
OOTA vs. ABP

### ISSUE THREAD

**Issue by stefanheinrichsen** [bitconnector/ttn_mapper_t-beam#4]
Just wanted to ask if there is any special reason for using ABP and not OOTA as activation method?

**Comment by bitconnector** [comment]
In order to use OOTA you have to be in a location with a good connection to ttn while startup. Thats mainly the rason for using ABP. Most of the mapping devices I've seen so far are using it, because the messages are getting through immediately.

I know about the security issues bundled with ABP and disabeling the frame counter check.

If you think it would be better to have the OOTA option I could add it to the program :)

**Comment by stefanheinrichsen** [comment]
Yes, I really think it would be beneficial. Also it will allow a more "out-of-the-box" usage of the mapper without diving into the last corner of the TTN v3 console.

If you add this activation it would be perfect. Else I will try to provide a patch during the next 1-2 weeks.
